# Open Sequence Spec

See how every sequencing goes.

# SRA quality check project:

## project summary 

  Check sequence quality of public NGSeq data, Don't wasting your time by downloading wrecked sequence.
  We offer statistics of quality value of NGSseq data, which is archived Sequence Read Archive.
  Sequence quality is calculated by FastQC, one of the best qual-check software.
  Details of FastQC is here (Thanks Simon!):
  http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc/

## browsing progress of this project

  Pre-calculated sequence quality can be browsed as a result of FastQC through our search&browse interface.
  NGSeq search-by-statistics service DBCLS SRAs:
    http://sra.dbcls.jp/
  or my pilot project which offers better browsing interface named 鎖鋸(kusari-noko):
    http://g86.dbcls.jp/kusarinoko

## other information

  main contributor of this project:
  Tazro Inutano Ohta, in Database Center for Life Science, Japan (DBCLS).
    http://g86.dbcls.jp/~iNut/

## ER

built for emergency, but always things happen here.

### what?

get files from SRA and dump it to fastq, then run FastQC to get read specs. formerly complicated, now it is just a loop of simple modules.

### where?

running on Rina Aizawa (DDBJ Super Computer System), in a node connected to the archive.

### who?

[me!](https://twitter.com/inutano)

### when?

since 2011, will end up with the end of next-generation.

### how?

read messy scripts or ask me

### lisence

"THE BEER-WARE LICENSE" (Rivision 42): [inutano@gmail.com](mailto:inutano@gmail.com) is contributing to this project. As long as you retain this notice you can do whatever you want with this stuff. If we meet some day, and you think this stuff is worth it, you can buy me a beer in return. Tazro Inutano Ohta

# Statistics Sequence Read Archive

Visualize Everything.

# Contact

Message to [inutano](http://twitter.com/inutano) or [Email me](mailto:t.ohta@dbcls.rois.ac.jp)
